### PR TITLE
add wattermark in UI

### PR DESCRIPTION
**Pull request by jayeshchoudhary**
image

### CODE REVIEW

**Comment by jayeshchoudhary** [comment]
@arshadkazmi42 can you please review

**Comment by jayeshchoudhary** [comment]
closes #23

File: src/components/HallOfFame.js [hunk 1]
```diff
@@ -33,6 +37,12 @@ function HallOfFame(props) {
             <CodeView>{hof}</CodeView>
           </Paper>
         </div>
+        <Typography color="textSecondary" className={classes.watermark}>
```
Comment (arshadkazmi42) [inline src/components/HallOfFame.js:40]: We want this to be part of the makrdown content. I think you can contact this using HTML into `hof` which is already getting passed to this component. We want the watermark to show up where ever the markdown is used.

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on this.

We want the watermark to be part of the `Markdown` section, so when someone uses the generated markdown, it will also show the watermark

**Comment by jayeshchoudhary** [comment]
@arshadkazmi42 updated.

**Review by arshadkazmi42 — APPROVED** [review]
LGTM 🎉

**Pull request merged**